mods_cocina_mappings/mods_to_cocina_subject.txt

```diff
@@ -120,9 +145,9 @@ Option B
 }
 
 6. Multi-term topic subject with authority for both set and terms
-<subject authority="lcsh" authorityURI="http://id.loc.gov/authorities/subjects/" valueURI="http://id.loc.gov/authorities/subjects/sh12345">
-  <topic authority="lcsh" authorityURI="http://id.loc.gov/authorities/subjects/" valueURI="http://id.loc.gov/authorities/subjects/sh23456">Horses</topic>
-  <topic authority="lcsh" authorityURI="http://id.loc.gov/authorities/subjects/" valueURI="http://id.loc.gov/authorities/subjects/sh34567">History</topic>
+<subject authority="lcsh" authorityURI="http://id.loc.gov/authorities/subjects" valueURI="http://id.loc.gov/authorities/subjects/sh12345">
+  <topic authority="lcsh" authorityURI="http://id.loc.gov/authorities/subjects" valueURI="http://id.loc.gov/authorities/subjects/sh23456">Horses</topic>
+  <topic authority="lcsh" authorityURI="http://id.loc.gov/authorities/subjects" valueURi="http://id.loc.gov/authorities/subjects/sh34567">History</topic>

```

Comment:
Typo in `valueURi`  also, shouldn't subjects uri have a trailing `/`?

Reply:
Fixed.